HW_36/usage_of_OOP.py

```diff
@@ -0,0 +1,173 @@
+from datetime import date, timedelta
+
+""" Абстрактный класс Person """
+
+
+class Person:
```

Comment:
As this class is abstract, it is better to inherit it from ABC to not allow instantiate from it